meta-flatpak/recipes-flatpak/flatpak-utils/files/flatpak-utils/Makefile.am

```diff
@@ -0,0 +1,32 @@
+SUBDIRS  = . src #orig
+doc_DATA = AUTHORS ChangeLog NEWS README
+
+# This is the only way with automake I know of to force 'check-git-hooks'
+# to be evaluated before 'all'. If there is a nicer way, I'm all ears...
+BUILT_SOURCES = install-git-hooks
```

Comment:
You don't need to install git hooks, if you have the source distribution inside the layer. There is no corresponding `.git` directory.